src/client/firebase/UserContext.js

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import React, { createContext, useContext, useEffect, useState } from 'react';
+import { useFirebase } from './FirebaseContext';
+
+const UserContext = createContext();
+
+export function UserProvider({ children }) {
+  const { isAuthenticated, getUserToken, authUser } = useFirebase();
+  const [user, setUser] = useState();
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function addUser() {
+      const token = await getUserToken();
+      console.log(token, 'context');
+      const response = await fetch('/api/users', {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+          'Content-type': 'application/json',
+          authorization: `Bearer ${token}`,
+        },
+        body: JSON.stringify(authUser),
```

Comment:
we need to send the uid. So authUser.uid

Reply:
But then how would the person working on the backend get the information of the user? (name and photoURL)

Reply:
Correct yes, we need at least the name also. Can the photo url be extracted from the authUser?

Reply:
yes, name, email, uid, and photoURL. Should I send it as an object with this info or send the authUser as it is?

Reply:
👌
Yes I see. I was actually wondering how to handle this. 
Better make an object then

Reply:
Then edit the users migration to add the photourl column